spec/data/changed_managed_files/opensuse131

```diff
@@ -1,6 +1,7 @@
 # Changed managed files [192.168.121.137] (2014-11-25 18:21:52)
 
   Files extracted: yes
+  * /opt/test-quote-char/link (link_path)
```

Comment:
Why is there a link without a single quote? What does it add to our test coverage?